src/Branching/effective-feature-isolation-on-tfvc.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Version Control - Effective feature isolation on TFVC
+description: Explore the feature isolation branching strategy and explore when and how to delete branches
+ms.assetid: 00303E47-5B7F-4A25-901B-0B6F2007911D
+ms.prod: vs-devops-almeffective-feature-isolation-on-rfvc/
+ms.technology: vs-devops-articles
+ms.manager: douge
+ms.date: 03/05/2017
+ms.author: willys
+author: willys
+---
+
+> 
+> #**THIS IS DRAFT.3 - WORK IN PROGRESS**
+> 
+
+#Effective feature isolation on TFVC
+
+Are you planning to implement Team Foundation Version Control‘s feature isolation strategy ([TFVC](https://msdn.microsoft.com/en-us/library/ms181237.aspx))? You probably have a few questions, such as:
+
+- Is it practical with large development teams?
+- Does it align well with your agile process(es)?
+- What’s the proper lifetime of isolated feature branches? 
+
+This topic aims to answer those questions, and give you a clear perspective on feature isolation in TFVC. (For guidance on TFVC branch strategies overall, read [Branching Strategies with TFVC](./branching-strategies-with-tfvc.md).) 
+
+> [!NOTE]
+> 
+> This topic covers TFVC. For Git, refer to [adopt a Git branching strategy](./git-branching-guidance.md). 
+
+#What's feature isolation?
+
+The feature isolation strategy allows you work on your features or fix bugs in **feature** branches (also known as “topic branches”) based off your **main** branch, or **dev**, or other central branches. It isolates your changes from the rest of your team, allows you to experiment with a feature, and enables easy rollback or merging of changes with your parent branch. 
```

Comment:
Perhaps add a note differentiating the "Feature" work item type. Some readers may think that this pattern is only used when delivering things from that middle backlog level. Don't change the name of the strategy, just mention that it's used when developing PBIs or User Stories or whatever CMMI calls them. :-)